docs/src/content/docs/ops/config/authentication.adoc

```diff
@@ -0,0 +1,52 @@
+---
+title: Authentication
+---
+
+== Server-level settings
+
+XTDB lets you configure which users can connect to the database and which type of authentication method they need to use.
+An authentication rule consists of a `user`, a `method` and an ip `address` (in IPv4 or IPv6 format).
+[source,yaml]
+----
+- user: admin
+  address: 127.0.0.1
+  method: PASSWORD
+----
+The above authentication rule means that the `admin` user requires a password connecting from `localhost`.
+The authentication rules are passed to the server at startup and can be configured as below. The rules are considered in order
+until the first one matches. If none matches the connection is rejected.
+
+Below is a pgwire server configuration with 3 different rules.
+
+[source,yaml]
+----
+server:
+  port: 5432
+  authn:
+   authnRules:
+      # admin always requires a password
+      - user: admin
+        method: PASSWORD
+      # We trust local connections
+      - address: 127.0.0.1
+        method: TRUST
+      # Everything else requires a password.
+      - method: PASSWORD
```

Comment:
worth double checking if this is the pg_hba behaviour - i.e. does it fall through if _either_ the rule doesn't match or the authn fails, or, if an earlier rule matches but the authn fails, does it then fail the whole authn regardless of later rules?

Reply:
From the docs
>There is no “fall-through” or “backup”: if one record is chosen and the authentication fails, subsequent records are not considered. If no record matches, access is denied.